user/gdm-settings/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "gdm-settings"
+pkgver = "4.4"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "blueprint-compiler",
+    "meson",
+    "gobject-introspection",
+]
+makedepends = [
+    "appstream-glib",
+    "desktop-file-utils",
+    "gettext",
+    "glib-devel",
+    "gtk4-devel",
+    "libadwaita-devel",
+    "python-gobject-devel",
+    "python-pkgconfig",
```

Comment:
host

Reply:
though i think this is not used? you only want regular `pkgconf` afaict, could be wrong just had a quick look